scripts/web/blueprints/grading.py

```
@@ -0,0 +1,692 @@
+    chroma_path = _shared_chroma_dir() if use_shared_chroma else run_root / "chroma_db"
+
+    from web.config import DEFAULT_LECTURE_CHUNKS
+    steps: list[dict] = []
+
+    # 1. Extract
+    code, out = _run(_cli() + [
+        "--mode", "extract",
+        "--data-dir", str(upload_dir),
+        "--output-root", str(OUTPUT_ROOT),
+        "--run-id", run_id,
+    ])
```

Ruff (0.15.11)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)